### ISSUE TITLE
Example will always override JSF options

### ISSUE THREAD

**Issue by zergeborg** [aleksandryackovlev/openapi-mock-express-middleware#42]
### Description on the bug
If the OpenAPI schema contains examples, the middleware will ALWAYS return it as part of the response. This very much prevents using the JSF library as intended. Even if I pass {useExamplesValue: false} to JSF, the middleware still uses the examples.

### Expected Behavior
If I pass {useExamplesValue: false} to JSF, the middleware should honor this setting.

### Actual Behavior
If I pass {useExamplesValue: false} to JSF, the middleware ignores it and it outputs the examples anyway.

### Code

```js
app.use(
  '/api', // root path for the mock server
  createMockMiddleware({
    spec: '/Users/openapi-en.yaml',
    options: {
        useExamplesValue: false,
    },
    configure: (jsf) => {
        jsf.option({
            useExamplesValue: false,
        });
    }
  }),
);
```

### How Do We Reproduce?
See above

**Comment by aleksandryackovlev** [comment]
https://github.com/aleksandryackovlev/openapi-mock-express-middleware/commit/953af410d18de15969a35146cea56285548836b8

**Comment by aleksandryackovlev** [comment]
Fixed in release #48
